Src/Support/Google.Apis.Auth/OAuth2/GoogleAuthConsts.cs

```diff
@@ -75,6 +75,21 @@ public static class GoogleAuthConsts
         /// <summary>Installed application localhost redirect URI.</summary>
         public const string LocalhostRedirectUri = "http://localhost";
 
+        /// <summary>IAM access token endpoint for service account.</summary>
+        internal const string IamServiceAccountEndpointCommonPrefix = "https://iamcredentials.googleapis.com/v1/projects/-/serviceAccounts/";
+
+        /// <summary>IAM access token endpoint for service account.</summary>
```

Comment:
For all of these that are format strings, that should be clear in the name and documentation. The "IAM accss token endpoint" doesn't include `{0}`, but a format string does.

Reply:
done